inc/data-structures.php

```diff
@@ -71,7 +75,7 @@ function register_profile() {
 				'ep_mask' => EP_AUTHORS,
 			],
 			'capability_type'     => 'post',
-			'map_meta_cap'        => null,
+			'map_meta_cap'        => false,
```

Comment:
WordPress accepts `null` or `false`, but this value is typecasted as a `boolean` in WordPress. So `null` gets a complaint from PHPStan.

Reply:
Changing into `false` does not add a regression since WordPress accepts `false` as well.